<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CentralWidget</class>
 <widget class="QWidget" name="CentralWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>多线程无边框UDP  demo</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="sideBarWidget" native="true">
     <property name="class" stdset="0">
      <string>SideBarWidget</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="groupButton1">
        <property name="text">
         <string>一级菜单</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="class" stdset="0">
         <string>GroupButton</string>
        </property>
        <property name="groupName" stdset="0">
         <string>one</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="itemButton1">
        <property name="text">
         <string>GGA</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="class" stdset="0">
         <string>GroupItemButton</string>
        </property>
        <property name="groupName" stdset="0">
         <string>one</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="itemButton2">
        <property name="text">
         <string>DHV</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="class" stdset="0">
         <string>GroupItemButton</string>
        </property>
        <property name="groupName" stdset="0">
         <string>one</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="itemButton3">
        <property name="text">
         <string>RMC</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="class" stdset="0">
         <string>GroupItemButton</string>
        </property>
        <property name="groupName" stdset="0">
         <string>one</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="itemButton4">
        <property name="text">
         <string>ECT</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="class" stdset="0">
         <string>GroupItemButton</string>
        </property>
        <property name="groupName" stdset="0">
         <string>one</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>写Udp测试</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="groupButton2">
        <property name="text">
         <string>网络</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="class" stdset="0">
         <string>GroupButton</string>
        </property>
        <property name="groupName" stdset="0">
         <string>two</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="itemButton15">
        <property name="text">
         <string>网络设置</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="class" stdset="0">
         <string>GroupItemButton</string>
        </property>
        <property name="groupName" stdset="0">
         <string>two</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="itemButton16">
        <property name="text">
         <string>连接</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="class" stdset="0">
         <string>GroupItemButton</string>
        </property>
        <property name="groupName" stdset="0">
         <string>two</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="itemButton17">
        <property name="text">
         <string>断开</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="class" stdset="0">
         <string>GroupItemButton</string>
        </property>
        <property name="groupName" stdset="0">
         <string>two</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="contentStackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="class" stdset="0">
      <string>ContentStackedWidget</string>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
